environments/network/preview.tfvars

```diff
@@ -0,0 +1,107 @@
+enable_debug = "true"
+
+network_address_space                  = "10.48.128.0/18"
+aks_00_subnet_cidr_blocks              = "10.48.128.0/20"
+aks_01_subnet_cidr_blocks              = "10.48.144.0/20"
+iaas_subnet_cidr_blocks                = "10.48.160.0/24"
+application_gateway_subnet_cidr_blocks = "10.48.161.0/25"
+
+additional_subnets = [
+]
+
+private_dns_subscription = "1baf5470-1c3e-40d3-a6f7-74bfbce4b348"
+private_dns_zones = [
+  "preview.platform.hmcts.net",
+  "service.core-compute-preview.internal",
+  "service.core-compute-idam-preview.internal",
+  "aat.platform.hmcts.net",
+  "service.core-compute-aat.internal",
+  "service.core-compute-idam-aat.internal",
+  "service.core-compute-idam-aat2.internal"
+]
+
+hub = "nonprod"
+
+additional_routes = [
+  {
+    name                   = "aat"
+    address_prefix         = "10.10.128.0/18"
+    next_hop_type          = "VirtualAppliance"
+    next_hop_in_ip_address = "10.11.8.36"
```

Comment:
This is not correct, it will cause an asymmetric route. Please review AAT routetable